### DISCUSSION TITLE
Can anyone help a Freeplane mind map user...not tech savvy.

### DISCUSSION THREAD

**Discussion by Sloanj44** [freeplane/freeplane#644] [Help]
I am an end user of FreePlane and have enjoyed it for a number of years.
I have downloaded the new version and something is dreadfully wrong.  I can't get the top menus to function.  They are gray and I seem to have a picture-in-picture view I can't get rid of.  
I have old files I'm unable to edit.
I'm not technically savvy but would like to return to using Freeplane.  I loved it.
Can anyone help lead me to a fix?

![image](https://user-images.githubusercontent.com/112812812/189338352-0fc1d774-a0af-4147-9ec5-3d001a633dad.png)

**Comment by Illegit** [comment]
Hi , I think you have activated the Viewing mode, go in the Edit Menu and click the Editing Mode.
illegit

**Comment by macmarrum** [comment]
Alternatively, you can run Command Search: `Ctrl+F1` by default
then type `Editing` to search for `Edit->Editing`
and hit `Enter` to activate

**Comment by euu2021** [comment]
> [...] and I seem to have a picture-in-picture view I can't get rid of. 

Do you mean, this?
![image](https://user-images.githubusercontent.com/77707706/189460291-e355d0df-5748-4a43-ab03-9bb7baa19ab8.png)

It's the map overview. You can right-click the background and turn it off here:

![image](https://user-images.githubusercontent.com/77707706/189460328-df9a96b4-2ef4-4765-b32b-a3676f9ced2c.png)


It also seems that your menubar disappeared? Again, toggle it by right-clicking the background:

![image](https://user-images.githubusercontent.com/77707706/189460759-9f8ed22b-4620-4417-ae58-58c3c7e0eb5f.png)



You may also want to try a smaller UI, and see if you like. Try changing these options:

![image](https://user-images.githubusercontent.com/77707706/189460612-1f7b8553-48b6-4f61-89f6-0f785bb37502.png)

Look how I use mine:

![image](https://user-images.githubusercontent.com/77707706/189460674-22ead9f0-3020-4564-85e2-c3ccf15576af.png)

**Comment by Sloanj44** [comment]
Yikes!  So simple...thank you, thank you. I'm back to it.
